code/modules/organs/organ_external.dm

```diff
@@ -596,7 +596,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		//Configurable regen speed woo, no-regen hardcore or instaheal hugbox, choose your destiny
 		heal_amt = heal_amt * config.organ_regeneration_multiplier
 		if(M_REGEN in owner.mutations)
-			heal_amt = heal_amt * 2 // Will make you regenerate twice as fast if you have regeneration
+			heal_amt = heal_amt * config.organ_regeneration_multiplier // Will heal based on organ_regeneration_multiplier twice if you have Regeneration superpower
```

Comment:
So it *squares* the healing rate? With default config that's the same as doubling it, but relying on that isn't great

Reply:
I could cook up a new config called regeneration multiplier and set that to multiply by 2

Reply:
You could but you could also just multiply it by two since that's what you intend to do anyway